openml/datasets/functions.py

```diff
@@ -897,6 +897,31 @@ def edit_dataset(
     return int(data_id)
 
 
+def fork_dataset(data_id) -> int:
+    """
+    Create a fork of the dataset with the calling authorized user as owner.
```

Comment:
Isn't it the authenticated user which becomes the new owner?

Reply:
Yes